src/lib/services/feature-toggle-service.ts

```diff
@@ -377,17 +377,53 @@ class FeatureToggleService {
             constraints: featureStrategy.constraints || [],
             parameters: featureStrategy.parameters,
             segments: segments.map((segment) => segment.id) ?? [],
+            sortOrder: featureStrategy.sortOrder,
         };
     }
 
     async updateStrategiesSortOrder(
         featureName: string,
+        environment: string,
+        project: string,
+        createdBy: string,
         sortOrders: SetStrategySortOrderSchema,
     ): Promise<Saved<any>> {
         await Promise.all(
-            sortOrders.map(async ({ id, sortOrder }) =>
-                this.featureStrategiesStore.updateSortOrder(id, sortOrder),
-            ),
+            sortOrders.map(async ({ id, sortOrder }) => {
+                const strategyToUpdate =
+                    await this.featureStrategiesStore.getStrategyById(id);
+                await this.featureStrategiesStore.updateSortOrder(
```

Comment:
can we return the updated strategy from this method so that we don't need to make another DB call on line 399?